config/routes.rb

```diff
@@ -644,6 +644,34 @@
       ) + adv_search_post + exp_post + save_post
     },
 
+    :container_build          => {
+      :get  => %w(
+        download_data
+        edit
+        index
+        new
+        show
+        show_list
+        tagging_edit
+        tag_edit_form_field_changed
+      ),
+      :post => %w(
+        button
+        create
+        dynamic_checkbox_refresh
```

Comment:
I don't know, to be honest I just copied this from persistent volumes. All those actions seem relevant but I am not sure how to test them.